appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -33,7 +33,10 @@ import com.appcues.data.model.styling.ComponentSize
 import com.appcues.data.model.styling.ComponentStyle
 import java.util.UUID
 
-internal fun Modifier.primitiveStyle(
+/**
+ * Used by ExperiencePrimitive Compose
+ */
+internal fun Modifier.outerPrimitiveStyle(
```

Comment:
had to break it down into outer and inner modifier, so we can fit in compositions in between (like backgroundImage)